.github/workflows/CI-time-tracker-ui.yml

```diff
@@ -48,11 +48,16 @@ jobs:
         STACK_EXCHANGE_ACCESS_TOKEN: ${{ secrets.STACK_EXCHANGE_ACCESS_TOKEN }}
         AZURE_APP_CONFIGURATION_CONNECTION_STRING: ${{ secrets.AZURE_APP_CONFIGURATION_CONNECTION_STRING }}
       run: |
-        chmod +x ./scripts/populate-var-file.sh
-        sh ./scripts/populate-var-file.sh
```

Comment:
if you are not using the populate-var-file.sh anymore, delete it from the project